core-services/testgrid-config-generator/_allow-list.yaml

```diff
@@ -36,20 +36,20 @@ periodic-ci-openshift-release-master-nightly-4.10-e2e-azure-csi-migration: broke
 periodic-ci-openshift-release-master-nightly-4.10-e2e-gcp-csi-migration: broken
 periodic-ci-openshift-release-master-nightly-4.11-e2e-aws-ovn-serial: informing
 release-openshift-origin-installer-e2e-azure-compact-4.4: broken
-periodic-ci-openshift-microshift-main-periodics-copr-rpm-health-check: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-api-machinery: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-apps: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-arch: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-auth: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-cli: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-instrumentation: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-network: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-node: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-scheduling: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-sig-storage: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-reboot: informing
-periodic-ci-openshift-microshift-main-periodics-rebase-on-nightlies: informing
-periodic-ci-openshift-microshift-main-periodics-test-srpm: informing
-periodic-ci-openshift-microshift-main-periodics-test-unit: informing
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-parallel: broken
-periodic-ci-openshift-microshift-main-periodics-e2e-openshift-conformance-serial: broken
+periodic-ci-openshift-microshift-main-periodics-copr-rpm-health-check: generic-informing
```

Comment:
this file doesn't have many entries in it.  How are other jobs included?

Reply:
No idea.  The approach here is what's recommended by CI [docs](https://docs.ci.openshift.org/docs/how-tos/add-jobs-to-testgrid/#adding-a-non-release-gating-job-to-testgrid).

Reply:
A better answer: openshift-ci runs a periodic testgrid config generator job that scans jobs, produces configs, and opens PRs into test-infra.  The tool is geared to place CI configs with ocp, origin, or release in the name are considered informing.  CI configs without those names default to generic-informing.

The bright side is that test-grid dashboard tabs can exist on more than one dashboard.  Once we merge this PR, we can open a PR upstream to copy the generated dashboard into the ocp-release-4.12-informing dashboard.

Reply:
Would it make sense to either add 'microshift' to the list of strings that tool looks for or add one of the existing strings to our job names? Maybe 'origin', since that's the more general "upstream" indicator, IIRC.

If we follow the current approach, would we have to have similar PRs to add our jobs to the 4.13 dashboard and others in the future?

Reply:
Unfortunately no.  So the tool looks for those substrings in test names and generates dashboard names from those.  The catch is in naming. There's a tight coupling, via this naming convention, that is present throughout the automation.  The test names must be in the pattern of `org-repo-branch-test-name`, and this pattern is passed through each layer of the config generators, from CI config -> Prow config -> TestGrid config). It makes it easy to ID at a glance what code base each job is testing.

But there is hope. First, tests can be represented on more than one dashboard at once.  It's just a matter of adding the dashboard-tab config to whichever dashboard _by hand._  So, if we tag our tests as generic-informing, they'll have a dashboard generated at https://testgrid.k8s.io/redhat-openshift-informing.  Then we open a PR in test-infra that just copies the generated dashboard-tabs into the release-ocp-4.12-informing dashboard.

I've gone around and around with the CI maintainers, and AFAICT, this looks like the least hacky workaround.